pkg/terminal/command.go

```diff
@@ -1537,24 +1540,32 @@ func examineMemoryCmd(t *Term, ctx callContext, args string) error {
 				"decimal":     'd',
 				"bin":         'b',
 				"binary":      'b',
+				"addr":        'a',
+				"address":     'a',
 			}
 			priFmt, ok = fmtMapToPriFmt[v[i]]
 			if !ok {
 				return fmt.Errorf("%q is not a valid format", v[i])
 			}
-		case "-len":
+		case "-count", "-len":
 			i++
 			if i >= len(v) {
-				return fmt.Errorf("expected argument after -len")
+				return fmt.Errorf("expected argument after -count")
 			}
 			var err error
-			length, err = strconv.Atoi(v[i])
-			if err != nil || length <= 0 {
-				return fmt.Errorf("len must be an positive integer")
+			count, err = strconv.Atoi(v[i])
+			if err != nil || count <= 0 {
+				return fmt.Errorf("count must be an positive integer")
```

Comment:
s/an positive/a positive/

Reply:
Thanks @aarzilli  for pointing out the problems.

The format of address is indeed hexidecimal. We can use `hexdecimal` to check that by `-fmt hex -size 8` or `-fmt hex -size 4`, to avoid the cases `a i386 core dump on a amd64 host`.

Reply:
done